pages/index.js

```diff
@@ -1,11 +1,47 @@
-import trips from "../public/trips";
+import initialTrips from "../public/trips";
 import styled from "styled-components";
 import Image from "next/image";
+import TripInputForm from "@/components/TripInputForm";
+import { useState } from "react";
+import { uid } from "uid";
 
 export default function HomePage() {
+  const [trips, setTrips] = useState(initialTrips);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const trip = Object.fromEntries(formData);
+
+    const startYear = startDate.value.substring(2, 4);
+    const startMonth = startDate.value.substring(5, 7);
+    const startDay = startDate.value.substring(8);
+    const formattedStartDate = `${startDay}/${startMonth}/${startYear}`;
+    const endYear = endDate.value.substring(2, 4);
+    const endMonth = endDate.value.substring(5, 7);
+    const endDay = endDate.value.substring(8);
+    const formattedEndDate = `${endDay}/${endMonth}/${endYear}`;
```

Comment:
you can write a little helper function called 'parseDate' to use it for the start and end date. By this, you don't multiply your code.